api/client/options.proto

```diff
@@ -106,7 +106,7 @@ message H1ConnectionReuseStrategy {
 
 // TODO(oschaaf): Ultimately this will be a load test specification. The fact that it
 // can arrive via CLI is just a concrete detail. Change this to reflect that.
-// highest unused number is 39
+// highest unused number is 40
```

Comment:
(pre-existing) Is this true? There seem to be id `105` just above it.

Reply:
added cc80ec1e